wolfcrypt/src/integer.c

```diff
@@ -3847,6 +3847,10 @@ int fast_s_mp_mul_high_digs (mp_int * a, mp_int * b, mp_int * c, int digs)
 #endif
   mp_word  _W;
 
+  if (a->dp == NULL) { /* JRB, avoid reading uninitialized values */
```

Comment:
If `a->dp == null` happened technically we need to call an `mp_grow(a)` here. During the refactor where I defer the allocation of `a->dp` this warning came up even though it doesn't happen in usage.